boards/seeed/xiao_esp32c6/xiao_esp32c6_hpcore.dts

```diff
@@ -37,6 +37,19 @@
 		watchdog0 = &wdt0;
 	};
 
+	rf_switch {
```

Comment:
The board initialization hook expects a label, so we should provide a label:
```diff
-	rf_switch {
+	rf_switch: rf_switch {
```
The current version never enables the external antenna since the label is missing.